src/templates/base/2019/contributors.html

```diff
@@ -404,8 +421,22 @@ <h1 class="title title-lg">
                 contributors.classList.remove(...teams);
                 contributors.classList.add(...filterTeams);
             }
+
+            for (index = 0; index < buttons.length; ++index) {
+              var button = buttons[index];
+              if (hash.includes(button.classList)) {
+                button.setAttribute('aria-pressed',true);
+              } else {
+                button.setAttribute('aria-pressed',false);
+              }
```

Comment:
Set the attribute's value directly to the boolean. Also compare using `className` (string) rather than `classList` (DOMTokenList).
```suggestion
              button.setAttribute('aria-pressed', hash.includes(button.className));
```